profiles/repositories.py

```diff
@@ -92,5 +96,8 @@ def next_id(self) -> str:
 
         return profile_id
 
+    def list(self, limit: int = None, offset: int = 0) -> List[Profile]:
+        return self.db.session.query(Profile).limit(limit).offset(offset).all()
```

Comment:
is there a default ordering? making it explicit here?

Reply:
Depends on #29.